lib/susedistribution.pm

```diff
@@ -742,6 +742,9 @@ sub activate_console {
         ensure_user($user);
         assert_screen(["text-logged-in-$user", "text-login"], 60);
         $self->set_standard_prompt($user, skip_set_standard_prompt => $args{skip_set_standard_prompt});
+        $serialdev = 'sshserial';
+        set_var('SERIALDEV', $serialdev);
```

Comment:
```suggestion
        set_var('SERIALDEV', 'sshserial');
```

and remove line above

Reply:
Yep. But I'd be really for fixing machines' setup and get rid of `use_ssh_serial_console`.

Reply:
This will not work, you have to change global variable $sshseriel, I started with your suggestion

Reply:
Having to set a global variable while there is openQA environment variable, huh :). I just love this mess :).

Reply:
not set, change ...